pages/k8s/ldap.md

```diff
@@ -197,6 +210,24 @@ juju config kubernetes-master keystone-policy=$(cat policy.yaml)
 
 The [default policy may be downloaded][policy] for easy editing.
 
+
+## Custom Certificate Authority
+
+When using a custom certificate authority attached to Keystone, some additional configuration is
+required.
+
+ * Add CA to client machines that will run `kubectl`.
+
+```
+sudo cp custom_ca.crt /usr/local/share/ca-certificates
+sudo update-ca-certificates
+```
+ * Add CA to the kubernetes-master configuration
+
+```bash
+juju config kubernetes-master keystone-ssl-ca="$(base64 custom_ca.crt)"
```

Comment:
This config value needs to be a path to the cert, not base64-encoded contents of the cert, right?

Reply:
@Cynerva not anymore. The path is something mapped into the container, which is impossible for an operator to do without altering the yaml. I have a config.yaml change for the master charm to update the information there to say it is base64 encoded.

Reply:
Yep, you're right, I misunderstood. Thanks.